packages/sources/por-address-list/src/endpoint/address.ts

```diff
@@ -23,6 +23,11 @@ export const inputParameters = new InputParameters({
     type: 'number',
     default: 10,
   },
+  batchGroupSize: {
+    description: 'The number of concurrent batched contract calls to make at a time',
+    type: 'number',
+    default: 10,
+  },
```

Comment:
I think we should make this an env var instead like `eth-beacon`. We don't necessarily need to hold all NOPs to the same group size through the job. Depending on the load their node can handle, they could adjust this number to get better performance.

Reply:
Sure, my concern was consistency since `eth-beacon` has both in its config, whereas `batchSize` is input param in por-address-list

Reply:
I was considering that too to at first to avoid confusing NOPs but I think we restrict `batchSize` through the request since it has more limitations than just the NOP's node's capabilities. So we hold all NOPs to the same value for the job. Since the group size is solely dependent on the NOPs, I think it makes sense to make this configurable on an individual level